webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-student-button.js

```diff
@@ -8,11 +8,24 @@ class SakaiRubricStudentButton extends RubricsElement {
 
     super();
 
+    this.hidden = true;
     this.instructor = false;
     this.forcePreview = false;
     this.i18nPromise = SakaiRubricsLanguage.loadTranslations();
   }
 
+  connectedCallback() {
```

Comment:
I'd use attributeChangedCallback here. That way, if some other js on the page set the attributes of the student button element, those changes would trigger a change in the hidden status. I know that's unlikely, but still ..